### PR TITLE
BLUE-1722 resources

### PR DESCRIPTION
**Pull request by coutinol**

### CODE REVIEW

**Comment by jerome-leanspace** [comment]
> @jerome-leanspace could you provide me (one more time ^^) the command to format the code?

For go:
`gofmt -w -s .`
`

For terraform:
`terraform fmt --recursive`

**Comment by coutinol** [comment]
> > @jerome-leanspace could you provide me (one more time ^^) the command to format the code?
> 
> For go: `gofmt -w -s .`
> 
> For terraform: `terraform fmt -recursive`

done

**Review by rohitdobariyaleanspace — APPROVED** [review]

**Review by elodietinland — APPROVED** [review]

File: services/activities/resource_functions/schemas.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+	"type": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.StringInSlice(validFormulaTypes, false),
+		Description:  helper.AllowedValuesToDescription(validFormulaTypes),
+	},
+}
+
+var dataSourceFilterSchema = map[string]*schema.Schema{
+	"ids": {
```
Comment (jerome-leanspace) [inline services/activities/resource_functions/schemas.go:89]: ids is already present in the default list of available filters so you don't have to put it here

File: services/resources/resources/schemas.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+		Type:     schema.TypeString,
+		Optional: true,
+	},
+	"default_level": {
+		Type:     schema.TypeFloat,
+		Optional: true,
+	},
+	"constraints": {
+		Type:     schema.TypeSet,
+		Optional: true,
```
Comment (jerome-leanspace) [inline services/resources/resources/schemas.go:51]: I think you should add `MaxItems: 4`

File: services/resources/resources/schemas.go [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+	},
+	"value": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: validation.FloatBetween(0.0, 360.0),
+	},
+}
+
+var dataSourceFilterSchema = map[string]*schema.Schema{
+	"ids": {
```
Comment (jerome-leanspace) [inline services/resources/resources/schemas.go:100]: same here

**Review by jerome-leanspace — COMMENTED** [review]
I guess I'm reviewing too late, I've left some comments

File: services/resources/resources/schemas.go (hunk 2 above)
Comment (coutinol) [reply services/resources/resources/schemas.go:51]: right, I'll create a bug fix for that

File: services/activities/resource_functions/schemas.go (hunk 1 above)
Comment (coutinol) [reply services/activities/resource_functions/schemas.go:89]: I feel like it's not the first you tell me this :-)

**Pull request merged**